### DISCUSSION TITLE
Can't access web interface code-server Nginx

### DISCUSSION THREAD

**Discussion by Techttv** [coder/code-server#4984] [Q&A]
Hi, I'm using code-server on Raspberry Pi 3B+ with raspbian lite (64 bit arm). I'm using swag(nginx on container) as proxy manager and all the other sites I'm hosting are working perfectly but code-server don't.  I start it with 0.0.0.0:4500 with the port 4500 forwarded to vscode.(subdomain).org. This is my console log
`pi@raspberrypi:~ $ code-server --bind-addr 0.0.0.0:4500                                                           
      [2022-03-12T08:40:56.177Z] info  code-server 4.1.0 9e620e90f53fb91338a2ba1aaa2e556d42ae52d5                        
      [2022-03-12T08:40:56.187Z] info  Using user-data-dir ~/.local/share/code-server                           
      [2022-03-12T08:40:56.311Z] info  Using config file ~/.config/code-server/config.yaml                              
      [2022-03-12T08:40:56.311Z] info  HTTPS server listening on https://0.0.0.0:4500/                              
      [2022-03-12T08:40:56.312Z] info    - Authentication is disabled                                        
      [2022-03-12T08:40:56.312Z] info    - Using certificate for HTTPS: ~/.local/share/code-server/localhost.crt`

Is still stuck right there and page is not working

**Comment by jsjoeio** [comment]
Hmm...I'm not sure if we can help you without more information unfortunately. What do you see when you navigate to the address? Any logs in the browser you can share?

**Reply by Techttv** [reply]
If I use the local-ip:port I can use it but If I try with the domain I set up with nginx the browser show me error 503. All the other sites works perfectly

**Reply by jsjoeio** [reply]
Sounds like it's related to your nginx config then. Do you mind posting that?

**Reply by Techttv** [reply]
I'm going to check if there's some personal infos and then I'm going to share it

**Reply by Techttv** [reply]
here you are
```
server{
    listen 443 ssl;
    listen [::]:443 ssl;
    server_name vscode.*;
    include /config/nginx/ssl.conf;
    client_max_body_size 0;
    location / {
      auth_basic "Restricted";
      auth_basic_user_file /config/nginx/.htpasswd;
      include /config/nginx/proxy.conf;
      include /config/nginx/resolver.conf;
      set $upstream_app 192.168.178.79;
      set $upstream_port 4500;
      proxy_set_header Upgrade $http_upgrade;
      proxy_set_header Connection upgrade;
      proxy_set_header Accept-Encoding gzip;
      set $upstream_proto http;
      proxy_pass $upstream_proto://$upstream_app:$upstream_port;
    }
  }
````

**Reply by jsjoeio** [reply]
Nothing on that sticks out to me 🤔  It might help to look for other [nginx examples](https://github.com/coder/code-server/discussions?discussions_q=nginx).

[This one](https://github.com/coder/code-server/discussions/2766#discussioncomment-406390) looks similar to your issue?
